src/de/willuhn/jameica/hbci/server/Converter.java

```diff
@@ -67,7 +67,8 @@ public static Umsatz HBCIUmsatz2HibiscusUmsatz(GVRKUms.UmsLine u) throws RemoteE
     umsatz.setPurposeCode(u.purposecode);
     umsatz.setEndToEndId(u.endToEndId);
     umsatz.setMandateId(u.mandateId);
-
+    umsatz.setCreditorId(u.other.creditorid);
+    
     //BUGZILLA 67 http://www.willuhn.de/bugzilla/show_bug.cgi?id=67
```

Comment:
Ohne diese Zeile wurde bei mir interessanterweise die creditorId nicht in hibiscus übernommen. Die andere Möglichkeit, es zu setzen ist der else-Zweig des if in Zeile 122:
`    if (u.isCamt && u.usage != null)`
Für den Fall, dass die Daten via CAMT kamen, muss also u.usage == null sein, damit der Else-Zweig genommen wird. Dies ist aber im CAMT-Fall eigentlich nicht der Fall. 

Die Abfrage weiter unten ab Zeile 170 kommt mir stattdessen komisch vor. Das entsprciht jetzt nicht mehr der Art, wie direkt darüber die Mandats-ID behandelt wird.